src/pages/RefundExternal.tsx

```diff
@@ -47,20 +58,46 @@ enum RefundType {
     Legacy,
 }
 
+const SWAPS_PER_PAGE = 10;
+
 const BtcLikeLegacy = (props: {
     refundJson: Accessor<SubmarineSwap | ChainSwap>;
     refundInvalid: Accessor<RefundError | undefined>;
 }) => {
     const { t } = useGlobalContext();
+    const { setRefundableUTXOs } = usePayContext();
 
     const [refundTxId, setRefundTxId] = createSignal<string>("");
 
+    const swap = createMemo(() => props.refundJson());
+
+    const [refundableSwap] = createResource(swap, async (swap) => {
+        if (swap) {
+            if (swap.type === SwapType.Submarine) {
```

Comment:
Same thing here. This also applies to chain swaps and should exclude swaps from RBTC